contracts/src/v0.8/dev/KeeperRegistrarDev.sol

```diff
@@ -302,6 +303,7 @@ contract KeeperRegistrarDev is TypeAndVersionInterface, ConfirmedOwner {
     uint256 amount,
     bytes calldata data
   ) external onlyLINK permittedFunctionsForLINK(data) isActualAmount(amount, data) isActualSender(sender, data) {
+    if (data.length < 292) revert InvalidDataLength();
```

Comment:
Note: This check runs after the actual assembly checks, so it's hard to add a test for this. (You'll have to pass amount, sender validation first to hit this error)

292 is the max length that we read (in isActualSender).